3_javascript/javascript1/week1/smart-ease.js

```diff
@@ -0,0 +1,82 @@
+console.log("Javascript/week1");
+//Age-ify (A future age calculator)
+const yearOfBirth = 1992;
+let yearFuture = 2024;
+const age = yearFuture - yearOfBirth;
+console.log(age);
+console.log("You will be " + age + " years old in " + yearFuture);
+
+//Goodboy-Oldboy (A dog age calculator)
+const dogYearOfBirth = 2017;
+let dogYearFuture = 2027;
+let dogYear = dogYearFuture - dogYearOfBirth;
+console.log(dogYear);
+let dogYearsCompareToHuman = dogYear * 7;
+console.log(dogYearsCompareToHuman);
+shouldShowResultInDogYears = true;
+shouldShowResultInDogYears = false;
+function test(shouldShowResultInDogYears) {
+    if (shouldShowResultInDogYears) {
+        return console.log("Your dog will be " + dogYear + " human years old in " + dogYearFuture);
+    }
+        return console.log("Your dog will be " + dogYearsCompareToHuman + " dog years old in " + dogYearFuture); 
+}
+
+test(true);
+test(false);
+
+//Housey pricey (A house price estimator)
+
+//House for Peter
+const housePriceForPeter = 2500000;
+let houseWideForPeter = 8;
+let houseDeepForPeter = 10;
+let houseHighForPeter =10;
+let houseGardenSizeForPeter = 100;
+const volumeInMetersForPeter = 8*10*10*2.5*1000;
```

Comment:
Hey I see that you have declared the house values, but you are not actually using it :) 

`const volumeInMetersForPeter = 8*10*10*2.5*1000;`

instead of this it should count also just the volumeInMeters so `*2.5*1000;` should not be part of this variable